DataStructures/HashTables/HashTable.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace DataStructures.HashTables
+{
+    public class HashTable<TValue> 
+    {
+        public HashTable(int buckets)
+        {
+            Buckets = buckets;
+            ResetBuckets();
+        }
+        public int Buckets { get; set; }
+        private static int BreakdownString(string key)
+        {
+            int x = 0;
+            for (int i = 0; i < key.Length; i++)
+            {
+                x = x + (int)key[i];
+            }
+            return x;
+        }
+        public static int GetHash(string key)
+        {
+            
+            int key_ascii = BreakdownString(key);
+            return key_ascii * 269;
+
+        }
+        public void Clear() => ResetBuckets();
+        private void ResetBuckets()
+        {
+            BucketList = new List<HashList<TValue>>();
```

Comment:
The `List()` constructor will let you specify the initial capacity:
```suggestion
            BucketList = new List<HashList<TValue>>(Buckets);
```